etcd/src/test/scala/io/buoyant/etcd/KeyTest.scala

```diff
@@ -409,12 +409,12 @@ class KeyTest extends FunSuite with ParallelTestExecution {
     val key = mkKey(base) {
       case (Method.Get, Watch(true, None)) if currentIndex < responses.length =>
         val idx = currentIndex.toInt
-        requested(idx).setDone()
-        responses(idx)
+        requested(idx.toInt).setDone()
+        responses(idx.toInt)
```

Comment:
I can drop it here, but otherwise:
```
[error] /Users/ver/b/l5d/etcd/src/test/scala/io/buoyant/etcd/KeyTest.scala:416: type mismatch;
[error]  found   : Long
[error]  required: Int
[error]         requested(idx).setDone()
[error]                   ^
[error] /Users/ver/b/l5d/etcd/src/test/scala/io/buoyant/etcd/KeyTest.scala:417: type mismatch;
[error]  found   : Long
[error]  required: Int
[error]         responses(idx)
[error]                   ^
[error] two errors found
[error] (etcd/test:compileIncremental) Compilation failed
[error] Total time: 10 s, completed May 2, 2016 7:49:22 PM
```